tempesta_fw/t/functional/helpers/control.py

```diff
@@ -326,11 +326,11 @@ def stop(self):
 
 class Nginx(object):
 
-    def __init__(self, listen_port, workers=1):
+    def __init__(self, listen_port, workers=1, ka=100):
```

Comment:
I don't see clear reason in this change. `Nginx.Config` class has method to configure keep alive settings, which requires _two_ parameters. Here we pass only one of them.

Reply:
fixed